modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -4,21 +4,21 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) is composed of a Salesforce organization with Community Cloud and a managed package that is installed in this organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization.
+Anypoint API Community Manager is composed of a Salesforce organization with Community Cloud and a managed package that is installed in this organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization.
 
-The ACM package is installed in an *independent* Salesforce organization from any other Salesforce organizations your company may already have, so that the API community data is isolated from your company operational data, such as ERP or CRM data.
+The API Community Manager package is installed in an *independent* Salesforce organization from any other Salesforce organizations your company may already have, so that the API community data is isolated from your company operational data, such as ERP or CRM data.
 
-The Salesforce organization with Community Cloud is provisioned through and included in the MuleSoft ACM license. You don't need to procure this Salesforce organization separately, and you cannot reuse existing Salesforce licenses for ACM. Find more information about ACM licenses in the xref:licensing-overview.adoc[Licensing Overview] section.
+The Salesforce organization with Community Cloud is provisioned through and included in the MuleSoft API Community Manager license. You don't need to procure this Salesforce organization separately, and you cannot reuse existing Salesforce licenses for API Community Manager. Find more information about API Community Manager licenses in the xref:licensing-overview.adoc[Licensing Overview] section.
 
-NOTE: Installing, updating or configuring ACM must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
+NOTE: Installing, updating or configuring API Community Manager must be done by a user account with the *System Administrator* profile in the API Community Manager Salesforce organization.
 
-To install and configure the ACM managed package, use the following preparation steps in this Salesforce organization.
+To install and configure the API Community Manager managed package, use the following preparation steps in this Salesforce organization.
```

Comment:
I suggest naming the steps rather than using "the following," because each is a separate topic, at increasing distance from this intro.

Reply:
Yes, that is better. Fixed.